src/components/NavLink.tsx

```diff
@@ -0,0 +1,27 @@
+import { FC, ReactNode } from "react";
+import { Link } from "react-router-dom";
+
+interface Props {
+  to: string;
+  title: string;
+  children: ReactNode;
+}
+
+const NavLink: FC<Props> = ({ to, title, children }) => {
+  // ignore last char to consider /instances and /instance as active
+  const matchPart = to.substring(0, to.length - 1);
+  const isActive = location.pathname.startsWith(matchPart);
```

Comment:
is it not better to use `useLocation` from `react-router` since it's better to subscribe to React's lifecycle via state variables?

Reply:
Yes, that's better -- I changed it.